upgrade web3

migration is still using a beta of web3.  it is falling behind the rest of the stack, which has moved on.  the old version of web3 has issues, like this during npm install:

> npm i @daostack/migration@0.0.1-rc.41-v1
npm WARN deprecated tar.gz@1.0.7: :warning:  WARNING :warning: tar.gz module has been deprecated and your application is vulnerable. Please use tar module instead:
npm WARN deprecated fs-promise@2.0.3: Use mz or fs-extra^3.0 with Promise Support
npm ERR! path /mnt/c/users/dkent/Documents/Projects/GitHub/DAOStack/client/node_modules/websocket
npm ERR! code EISGIT
npm ERR! git /mnt/c/users/dkent/Documents/Projects/GitHub/DAOStack/client/node_modules/websocket: Appears to be a git repo or submodule.
npm ERR! git     /mnt/c/users/dkent/Documents/Projects/GitHub/DAOStack/client/node_modules/websocket

--- a/package.json
+++ b/package.json
@@ -49,7 +49,7 @@
     "@daostack/arc-hive": "0.0.1-rc.4",
     "ethereumjs-wallet": "^0.6.3",
     "fstream": "^1.0.12",
-    "ganache-cli": "6.7.0",
+    "ganache-cli": "6.9.0",
     "hdwallet-accounts": "0.0.1",
     "inquirer": "^6.2.2",
     "jsonschema": "^1.2.4",
@@ -59,7 +59,7 @@
     "prompt-confirm": "^2.0.4",
     "rimraf": "^2.6.3",
     "tar": "^4.4.8",
-    "web3": "1.0.0-beta.37",
+    "web3": "1.2.6",
     "yargs": "^12.0.2"
   },
   "devDependencies": {

